### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by charmcity-code**
add spoiler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,7 @@ original story he is never called an egg, we made that up.
 
 An egg probably shouldn't sit on a wall, given its shell and all, what 
 a dangerous fall.
+
+## Spoiler Alert
+
+Humpty fell and Lincoln ate him.
```
Comment (leafoflegend) [inline README.md]: Fell off what? Who is Lincoln?

**Review by leafoflegend — CHANGES_REQUESTED** [review]
Missing pivotal details.

File: README.md (hunk 1 above)
Comment (charmcity-code) [reply README.md]: good point... i will update

**Review by leafoflegend — APPROVED** [review]
The story is tragic, but moving.

**Pull request merged**